lib/dev-server.js

```diff
@@ -30,7 +32,11 @@ const port = process.env.TEST_PORT ?? 3010;
   });
 
   app.get('/', async (req, res) => {
-    const output = renderPageList(pages, title);
+    const nunjucksEnvironment = createNunjucksEnvironment();
+    const PROJECT_PATH = process.cwd();
+    const pageSearch = renderSearch(`${PROJECT_PATH}/templates/layout/_dsTemplate.njk`, nunjucksEnvironment);
+    const output = renderPageList(pages, title, pageSearch);
+    await buildSearchIndex();
```

Comment:
Config for the search should probably be added in config.indexPage.js because that is where the rest of the page is configured